src/plugins/intel_cpu/src/nodes/executors/shl/shl_eltwise.cpp

```diff
@@ -0,0 +1,226 @@
+#include "shl_eltwise.hpp"
+#include "shl_utils.hpp"
+#include "csinn/csi_nn.h"
+#include "utils/debug_capabilities.h"
+
+namespace ov {
+namespace intel_cpu {
+
+inline void log_unsupported_prec(const std::vector<MemoryDescPtr>& srcDescs,
+                                 const std::vector<MemoryDescPtr>& dstDescs,
+                                 const Algorithm eltwiseAlgorithm) {
+    std::string srcPrec;
+    for (size_t i = 0; i < srcDescs.size(); i++) {
+        srcPrec += srcDescs[i]->getPrecision().to_string() + " ";
+    }
+    DEBUG_LOG(algToString(eltwiseAlgorithm), ": provided combination of src precisions: [", srcPrec,
+                          "] and dst precision: ", dstDescs[0]->getPrecision().to_string(), " is not supported");
+}
+
+bool ShlEltwiseExecutor::isEltwiseAlgorithmSupported(Algorithm algorithm) {
+    if (one_of(algorithm, Algorithm::EltwiseAdd,
+                          Algorithm::EltwiseSubtract,
+                          Algorithm::EltwiseMultiply,
+                          Algorithm::EltwiseDivide,
+                          Algorithm::EltwiseMaximum,
+                          Algorithm::EltwiseMinimum,
+                          Algorithm::EltwiseExp,
+                          Algorithm::EltwiseRelu,
+                          Algorithm::EltwisePrelu)) {
+        return true;
+    }
+    return false;
+}
+
+bool ShlEltwiseExecutorBuilder::isSupported(const EltwiseAttrs& eltwiseAttrs,
+                                            const std::vector<MemoryDescPtr>& srcDescs,
+                                            const std::vector<MemoryDescPtr>& dstDescs) const {
+    auto checkPrecision = [&srcDescs, &dstDescs](std::vector<ov::element::Type> srcVecPrc, ov::element::Type dstPrc) -> bool {
+        for (size_t i = 0; i < srcDescs.size(); i++) {
+            if (srcDescs[i]->getPrecision() != srcVecPrc[i]) return false;
+        }
+        if (dstDescs[0]->getPrecision() != dstPrc) { return false; }
+        return true;
+    };
+
+    switch (eltwiseAttrs.algorithm) {
```

Comment:
If changed in this way, some tests will fail with the reason `Actual: ref_f32, expected: shl_f32`. As far as I see, there are no obvious differences between the two implementation. Im trying to figure it out later.

Reply:
Seems like I just missed add '!' in condition. Because my suggested condition looks like: if all precisions are supported, executor doesn't support this configuration. For sure, it's incorrect 😄 
Just try to add negation to the condition please and I think it will work.

Reply:
Thanks for the reminder! It is really hard to find a `!` problem! I will try it now!